docs/topics/native/native-cocoapods-dsl-reference.md

```diff
@@ -118,15 +118,21 @@ kotlin {
 The `pod()` function call adds a CocoaPods dependency to the Pod built from this project. Each dependency requires
 a separate function call.
 
-You can specify the name of a Pod library in the function parameters and additional parameter values, like the `version` and
-`source` of the library, in its configuration block.
+You can specify the name of a Pod library in the function parameters and additional parameter values, like the `version`
+and `source` of the library, in its configuration block.
 
-| **Name**      | **Description**                                                                                                                                                                                                                                                                                                                                             | 
-|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `version`     | The library version. To use the latest version of the library, omit the parameter.                                                                                                                                                                                                                                                                          |
+In the function parameters, you can specify the name of a Pod library and the `linkOnly` option to use pod dependencies
+with dynamic frameworks only for linking without generating C-interop bindings.
+
+In the configuration block, you can add parameter values, like the `version` and `source` of the library: 
```

Comment:
Is this a duplicate of line 122?

Reply:
True, I removed the duplicated text and simplified the whole desciption